src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_issue_name!(name)
```

Comment:
uh, two negations and how do you extend this?

(tracker.is_cve? && Regexp.new(/^\d\d\d\d-\d+$/).match(name)) || (tracker.is_bugzilla? && (not tracker.name == "exception)) && ...)....

I do really prefer a flow control here.

I can add the is_cve? though...